supertokens_python/recipe/thirdparty/providers/github.py

```diff
@@ -95,14 +95,11 @@ async def validate_access_token(
         "Authorization": f"Basic {basic_auth_token}",
         "Content-Type": "application/json",
     }
-    payload = json.dumps({"access_token": access_token})
-
-    resp = requests.post(url, headers=headers, data=payload)
 
-    if resp.status_code != 200:
+    try:
+        body = await do_post_request(url, {"access_token": access_token}, headers)
```

Comment:
change this function to return a status code too, and check for that

Reply:
Done